EncoreTickets.SDK.Tests/Tests/Api/ApiClientWrapperBuilderTests.cs

```diff
@@ -63,7 +63,7 @@ internal class ApiClientWrapperBuilderTests
                 RequestMethod.Post,
                 new Dictionary<string, string>
                 {
-                    {"x-SDK", "EncoreTickets.SDK.NET"},
+                    {"x-SDK", "EncoreTickets.SDK.NET 1.0.0"},
                     {"x-apply-price-engine", "true"},
```

Comment:
hi, you can remove these now as they are no longer required

{"x-apply-price-engine", "true"},
{"x-market", "broadway"},